### PR TITLE
remove support for CUDA 7.0

### PR DESCRIPTION
**Pull request by BenjaminW3**
Currently only for testing.

### CODE REVIEW

File: include/alpaka/vec/Vec.hpp [hunk 1]
```diff
@@ -46,13 +46,6 @@
 #include <type_traits>                      // std::enable_if, std::decay, std::is_same
 #include <algorithm>                        // std::min, std::max, std::min_element, std::max_element
 
-// Some compilers do not support the out of class versions:
-// - the nvcc 7.0 CUDA compiler
```
Comment (BenjaminW3) [inline include/alpaka/vec/Vec.hpp]: Testing showed, that this work around is required even for CUDA 7.5 and CUDA 8.0...

Comment (psychocoderHPC) [inline include/alpaka/vec/Vec.hpp]: Than we should only update the comment

Comment (BenjaminW3) [inline include/alpaka/vec/Vec.hpp]: This comment has already been added in a different PR.

**Comment by BenjaminW3** [comment]
For now I do not see a hard reason to remove CUDA 7.0.
I could not find / remember places where CUDA 7.0 required workarounds due to compiler bugs.
The C++ support should be nearly identical for all 3 current versions.
Feature wise  CUDA 7.5 only brought experimental support for GPU lambdas and CUDA 8.0 brought support for heterogeneous lambdas. Both are not required by alpaka.
Does anybody else see a reason to remove CUDA 7.0 support? Else, I would close this Pull Request for now and keep it in the list of supported CUDA versions.

**Pull request closed without merging**